src/reuse/modules/mobile/Mobile.ts

```diff
@@ -0,0 +1,13 @@
+import gestures, { Gestures } from "./gestures";
+import element, { ElementModule } from "./element";
+import userInteraction, { UserInteraction } from "./userInteraction";
+import device, { Device } from "./device";
+
+export class Mobile {
+  gestures: Gestures = gestures;
+  userInteraction: UserInteraction = userInteraction;
+  element: ElementModule = element;
```

Comment:
@Templada Previous comment in #287 
is 'Element' protected?

Reply:
it's just like same web, /Users/i542683/GitHub/qmate_bma_17092024/wdio-qmate-service/src/reuse/modules/nonUi5/element.ts